apcd-cms/src/apps/exception/templates/exception_submission_form/exception_other_form.html

```diff
@@ -124,9 +121,25 @@ <h4>Acknowledgment of Terms</h4>
         </button>
       </div>
       </form>
+      {# Footnotes #}
+      <div class="o-section o-section--style-light">
+        <hr />
+        <small>
+          <sup>*</sup> Exceptions cannot be granted for periods longer than a year<br />
+          <sup>**</sup>Exceptions cannot be granted "from any requirement in insurance code Chapter 38"<br />
+        </small>
+        <hr />
+      </div>
       <div class="form-success" style="display: none">
         <p>Thank you for your request.</p>
       </div>
+      {% else %}
+      <p class="c-message c-message--error">
```

Comment:
I think you can put this dashboard link in a separate template and load it in each of these forms instead.

Reply:
Added a template for a user with no sub id rather than displaying the form. Great suggestion learned a lot about Django views, classes, and methods!